features/steps/steps.go

```diff
@@ -39,17 +44,76 @@ func (c *PopulationTypesComponent) cantabularIsUnresponsive() error {
 	return nil
 }
 
-func (c *PopulationTypesComponent) theServiceRespondsWithAnInternalServerErrorSaying(expected string) error {
+func (c *PopulationTypesComponent) theServiceRespondsWithAnInternalServerErrorSaying(expectedHttpCode int, expected string) error {
 	resp := c.apiFeature.HttpResponse
-	if resp.StatusCode != http.StatusInternalServerError {
+	if resp.StatusCode != expectedHttpCode {
```

Comment:
Doesn't the check for http status code come built in to api.feature? Also the http response itself?